### PR TITLE
use different icon when station is full

### PR DESCRIPTION
**Pull request by myin142**
I think this feature might be useful if you want to know if there is space left in a station without always clicking on the icon. At least for me it would be very helpful. I have also read some reviews on the play store that want this feature.

### CODE REVIEW

File: src/components/Plan.js [hunk 1]
```diff
@@ -62,7 +62,9 @@ class Plan extends Component {
                 latitude: station.latitude,
                 longitude: station.longitude,
               }}
-              image = {station.free_bikes == 0 ? require('../../assets/img/bikes_empty.png') : require('../../assets/img/citybike_logo.png')}
+              image = {station.free_bikes == 0 ? require('../../assets/img/bikes_empty.png')
```
Comment (tomekz) [inline src/components/Plan.js]: The nested ternary is not very readable. There is even a [ESLint rule for that](https://eslint.org/docs/rules/no-nested-ternary) 
I would suggest to extract the logic to a seperate function on the same component level and use simply if blocks

File: src/components/StationCard.js [hunk 2]
```diff
@@ -22,7 +22,9 @@ class StationCard extends Component {
               containerStyle = {styles.cardAvatar}
               small
               rounded
-              source={station.free_bikes == 0 ? require('../../assets/img/bikes_empty.png') : require('../../assets/img/citybike_logo.png')}
+              source = {station.free_bikes == 0 ? require('../../assets/img/bikes_empty.png')
```
Comment (tomekz) [inline src/components/StationCard.js]: Same bout the nestet ternaries here

**Review by tomekz — CHANGES_REQUESTED** [review]
Hey, first of all thanks for your very useful contribution !
In general all looks good,  I like the new icons and have only small refactoring request you'll find in the comments.

**Comment by myin142** [comment]
I have now moved it to a separate function.

**Review by tomekz — APPROVED** [review]

**Pull request merged**